tests/test_urldispatch.py

```diff
@@ -1151,7 +1151,7 @@ def test_invalid_route_name(router: Any) -> None:
         router.add_get("/", make_handler(), name="invalid name")
 
 
-def test_invalid_route_name(router) -> None:
+def test_invalid_route_name_identifier(router) -> None:
```

Comment:
Could you parametrize the previous test instead? It's basically the same with just one thing being different.